core/src/test/java/org/elasticsearch/node/NodeTests.java

```diff
@@ -241,6 +242,34 @@ public void testDefaultPathDataNotSet() throws IOException {
         }
     }
 
+    public void testTribeNodeDeprecation() throws IOException {
```

Comment:
Well, you right, I did this test because I moved the deprecation logger in the start method of the service which is not necessary as in the constructor there already is a check whether or not tribe node is enable. I moved the log back. I'm writing a tests, running checks, and I will push it.